projects/components/src/select/select.component.ts

```diff
@@ -34,18 +34,41 @@ import { SelectSize } from './select-size';
         this.groupPosition,
         selected ? selected.style.toString() : '',
         this.showBorder ? 'border' : '',
-        this.disabled ? 'disabled' : ''
+        this.disabled ? 'disabled' : '',
+        this.triggerDisplayMode
```

Comment:
Is this needed? Can we contain the display mode styling fully within the trigger switch?